workflows/diagnostics/fv3net/diagnostics/offline/_helpers.py

```diff
@@ -222,3 +226,41 @@ def insert_column_integrated_vars(
         ds = ds.assign({column_integrated_name: da})
 
     return ds
+
+
+@curry
+def coarsen_cell_centered(
+    ds: xr.Dataset, coarsening_factor: int, weights: xr.DataArray
+) -> xr.Dataset:
+    return vcm.cubedsphere.weighted_block_average(
+        ds.drop_vars("area", errors="ignore"),
+        weights=weights,
+        coarsening_factor=coarsening_factor,
+        x_dim="x",
+        y_dim="y",
+    )
+
+
+def res_from_string(res_str: str) -> int:
+    if res_str.lower().startswith("c"):
+        res = ""
+        for c in res_str[1:]:
+            if c.isnumeric():
+                res += c
+            else:
+                break
```

Comment:
Oh, I see.. the test grid is `c8_...`. It might just be clearer going forward if that entry gets renamed in the catalog to `c8` and find all and replace  the references in the tests to just `c8`.

Reply:
Hmm, I'm hesitant to rename things in the catalog, but then again that one is presumably only used here.

Reply:
OK the situation was actually more complicated but also easier to deal with than I realized. Because the test now uses `needs_grid=False` it's not reading anything from the catalog, as the grid is supplied via synth. So we can change the test resolution to `c8` and leave the catalog alone; the fv3fit regression tests use the catalog's `c8_random_values` grid so that will stay as is.